test/e2e/specs/editor/various/navigable-toolbar.spec.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Block Toolbar', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.describe( 'Contextual Toolbar', () => {
+		test( 'should not scroll page', async ( { page, pageUtils } ) => {
+			while (
+				await page.evaluate( () => {
+					const scrollable = window.wp.dom.getScrollContainer(
+						document.activeElement
+					);
+					return ! scrollable || scrollable.scrollTop === 0;
+				} )
+			) {
+				await page.keyboard.press( 'Enter' );
+			}
+
+			await page.keyboard.type( 'a' );
+
+			const scrollTopBefore = await page.evaluate(
+				() =>
+					window.wp.dom.getScrollContainer( document.activeElement )
+						.scrollTop
+			);
+			await pageUtils.pressKeys( 'alt+F10' );
+			await expect(
+				page
+					.getByRole( 'toolbar', { name: 'Block Tools' } )
+					.getByRole( 'button', { name: 'Paragraph' } )
+			).toBeFocused();
+
+			const scrollTopAfter = await page.evaluate(
+				() =>
+					window.wp.dom.getScrollContainer( document.activeElement )
+						.scrollTop
+			);
+
+			expect( scrollTopBefore ).toBe( scrollTopAfter );
+		} );
+
+		test( 'navigates into the toolbar by keyboard (Alt+F10)', async ( {
```

Comment:
@jeryj mentioned that this is already covered in another test. We can remove it and leave only `should not scroll page`.

https://github.com/WordPress/gutenberg/blob/4d0f303bf0bf231db12221f03548cbc067c302b1/test/e2e/specs/editor/various/shortcut-focus-toolbar.spec.js#L12

P.S. There's also a merge conflict that needs to be resolved.

Reply:
I see conflicts when I try to rebase this branch on top of the trunk, but I can't push on this branch.

Creating a new PR with all the feedback applied might be easier.